frontend/src/constants/routes.js

```diff
@@ -38,7 +38,10 @@ export const CONTACT_HOME_PAGE = {
 
 export const MINE_SUMMARY = {
   route: "/dashboard/:id/:activeTab",
-  dynamicRoute: (id, activeTab = "summary") => `/dashboard/${id}/${activeTab}`,
+  dynamicRoute: (id, activeTab = "summary", filterParams) =>
```

Comment:
Even though it's a bit of a pain right now, I think this should be updated to accept an object of params instead of adding another argument to the function. `CONTACT_HOME_PAGE` and `MINE_HOME_PAGE` above do this well in the `.dynamicRoute()`.

I think we're getting dangerously close to unmanageable overloading of this function by adding a third argument and a client-side specific route to the server-side route building logic.

Happy to chat about it if you have any questions

Reply:
Follow-up: I just realized this doesn't build an API route. It's another clientside route. My first paragraph still applies. The second one does not

Reply:
This route is following similar logic to `CONTACT_HOME_PAGE` and `MINE_HOME
_PAGE`
The `filerParams` are an object and being handled the same way, the only difference with this route is that has an `id` whereas the others do not.

Reply:
Chatted about this IRL. Marking as resolved